README.md

```diff
@@ -41,19 +41,18 @@ NebulaClientOptions nebulaClientOptions = new NebulaClientOptions.NebulaClientOp
 NebulaGraphConnectionProvider graphConnectionProvider = new NebulaGraphConnectionProvider(nebulaClientOptions);
 NebulaMetaConnectionProvider metaConnectionProvider = new NebulaMetaConnectionProvider(nebulaClientOptions);
 
-ExecutionOptions executionOptions = new VertexExecutionOptions.ExecutionOptionBuilder()
+VertexExecutionOptions executionOptions = new VertexExecutionOptions.ExecutionOptionBuilder()
                 .setGraphSpace("flinkSink")
                 .setTag("player")
                 .setIdIndex(0)
                 .setFields(Arrays.asList("name", "age"))
                 .setPositions(Arrays.asList(1, 2))
-                .setBatch(2)
-                .builder();
+                .setBatchSize(2)
```

Comment:
Thanks for the suggestion! Yeah I think it's a good idea to ensure compatibility of user-facing interfaces within the same major version.

I've added back the methods such as `.setBatch()` and `.builder()` and marked them as `@Deprecated`, while in `README.md` I use the new methods. Hopefully this can encourage users to move to the new methods, without breaking existing code. Does this approach look fine to you? @Nicole00

Reply:
> Hopefully this can encourage

Great approach, thanks for changing it.